### PR TITLE
Gui iteration2

### PR DESCRIPTION
**Pull request by DaultonLeach**
GUI is Good for iteration 2. The variables get passed to the driver for all basic variables. Random tests are also working in regards to sending the data to the driver.

### CODE REVIEW

**Review by danielway — CHANGES_REQUESTED** [review]
Looks like there are style errors.

File: src/test/java/edu/iastate/ece/sd/sdmay2126/runner/selenium/FBASeleniumOutputCollectorTest.java [hunk 1]
```diff
@@ -45,16 +46,22 @@ public void setUp() {
 
     @Test
     public void collectorWillReturnDefaultOfNegOneObjectiveValueIfLabelNotFound() {
-        Job job = new Job(new FBAParameters(false, false, false));
-        FBAOutput result = (FBAOutput) classToTest.collectOutput(job, mockFBACard);
+        Job job = new Job(new FBAParameters(false, false, false,
+                0, 0, 0, 0, 0,
+                0, 0, 0));
+        FBAOutput result = (FBAOutput) classToTest.collectOutput(job);
+       // FBAOutput result = (FBAOutput) classToTest.collectOutput(job, mockFBACard);
```
Comment (danielway) [inline src/test/java/edu/iastate/ece/sd/sdmay2126/runner/selenium/FBASeleniumOutputCollectorTest.java:53]: Why did you comment the second parameter `mockFBACard`?

I can build this successfully with Maven, but when I try to run in Intellij it I get errors from these lines. That seems like a potential problem with our Maven pipeline. @jakedv24

Comment (DaultonLeach) [reply src/test/java/edu/iastate/ece/sd/sdmay2126/runner/selenium/FBASeleniumOutputCollectorTest.java]: Swapped the second and first parameters Daniel, I thought it was outdated originally, but we do need that in fact. These changes should be done now

**Review by danielway — APPROVED** [review]
Looks good to me @DaultonLeach, thanks for making those changes.

**Pull request merged**